cmd/crc/cmd/daemon_linux.go

```diff
@@ -1,16 +1,88 @@
 package cmd
 
 import (
+	"fmt"
 	"net"
 	"os"
 
 	"github.com/code-ready/crc/pkg/crc/constants"
 	"github.com/code-ready/crc/pkg/crc/logging"
 	"github.com/code-ready/gvisor-tap-vsock/pkg/transport"
+
+	"github.com/coreos/go-systemd/activation"
+	"github.com/mdlayher/vsock"
+)
+
+const (
+	vsockUnitName = "crc-vsock.socket"
+	httpUnitName  = "crc-http.socket"
 )
 
+var systemdListeners map[string][]net.Listener
+
+func init() {
+	// listenerWithNames() cannot be called multiple times
+	systemdListeners, _ = listenersWithNames()
+}
```

Comment:
> Also we are not consuming the error from listenersWithNames() so why we need to return it if it's value always going to be nil?

Since it's code c&p'ed from https://github.com/coreos/go-systemd/blob/f9f254610339c14ad7c50e9352848c7ff62fc81f/activation/listeners.go#L42 I'd prefer to avoid having too many differences between the 2 implementations

> instead of using init why don't we just use the var systemdListeners to assign return value of listenersWithNames() function?
Ah, thanks for the suggestion, I did not think/know this would compile just fine:
```
var systemdListeners, _ = listenersWithNames()
```